src/actions.ml

```diff
@@ -510,79 +536,95 @@ let minimize_failed_test ~bot_info ~owner ~repo ~pr_id ~base ~head
-            if String.is_suffix ~suffix:".v" filename then
-              match
-                ( extract_artifact_url build_job base_pipeline_summary
-                , extract_artifact_url build_job head_pipeline_summary
-                , extract_artifact_url name base_pipeline_summary
-                , extract_artifact_url name head_pipeline_summary )
-              with
-              | ( Some base_build_url
-                , Some head_build_url
-                , Some base_job_url
-                , Some head_job_url ) ->
-                  run_ci_minimization ~bot_info ~comment_thread_id:pr_id ~owner
-                    ~repo ~docker_image ~target ~opam_switch
-                    ~failing_urls:(head_build_url ^ " " ^ head_job_url)
-                    ~passing_urls:(base_build_url ^ " " ^ base_job_url)
-                    ~base ~head
-              | None, _, _, _ ->
-                  Lwt_io.printf
-                    "Couldn't find base build job url for %s in:\n%s\n"
-                    build_job base_pipeline_summary
-              | _, None, _, _ ->
-                  Lwt_io.printf
-                    "Couldn't find head build job url for %s in:\n%s\n"
-                    build_job head_pipeline_summary
-              | _, _, None, _ ->
-                  Lwt_io.printf "Couldn't find base job url for %s in:\n%s\n"
-                    name base_pipeline_summary
-              | _, _, _, None ->
-                  Lwt_io.printf "Couldn't find head job url for %s in:\n%s\n"
-                    name head_pipeline_summary
-            else
-              Lwt_io.printf
-                "Not a Coq failure for job %s (%s is not a Coq file): not \
-                 minimizing.\n"
-                name filename
-          else
-            Lwt_io.printf
-              "Couldn't find an error message for job %s in details text:\n%s\n"
-              name text
+          let missing_error, non_v_file =
+            if string_match ~regexp:"\nFile \"\\([^\"]*\\)\"" text then
+              let filename = Str.matched_group 1 text in
+              ( false
+              , if String.is_suffix ~suffix:".v" filename then None
+                else Some filename )
+            else (true, None)
+          in
+          match
+            ( extract_artifact_url build_job base_pipeline_summary
+            , extract_artifact_url build_job head_pipeline_summary
+            , extract_artifact_url name base_pipeline_summary
+            , extract_artifact_url name head_pipeline_summary )
+          with
+          | ( Some base_build_url
+            , Some head_build_url
+            , Some base_job_url
+            , Some head_job_url ) ->
+              Ok
+                ( { base_job_failed
+                  ; base_job_errored
+                  ; missing_error
+                  ; non_v_file
+                  ; job_kind
+                  ; head_job_succeeded
+                  ; overlayed= false (* XXX FIXME *) }
```

Comment:
This is non-trivial to fetch this info. Probably worth delaying to another PR, especially given that the way the `@coqbot: merge now` feature gets this info is also incomplete (and we would like it to retrieve the content of the file rather than the filename). In the meantime, maybe remove this field to not do as if we had this info.

Reply:
I've commented it out